ui/raidboss/data/05-shb/alliance/the_copied_factory.js

```diff
@@ -504,13 +440,7 @@
       condition: function(data, matches) {
         return data.me == matches.target;
       },
-      infoText: {
-        en: 'Tank Buster on YOU',
-        de: 'Tankbuster auf DIR',
-        fr: 'Tankbuster sur VOUS',
-        cn: '死刑点名',
-        ko: '탱버 대상자',
-      },
+      response: Responses.tankBuster(),
```

Comment:
I guess the condition makes this only on you.  This should be Responses.tankBuster('info') though to match previous text? I don't think this is super serious.